File: bacteria1.java
import java.util.Scanner;

public class bacteria1{


    public static void main(String[] args){

        Scanner scanner= new Scanner (System.in);


        String entrada= scanner.nextLine();
        String comparacao= scanner.nextLine();

        if (entrada.contains(comparacao)){

            System.out.println ("Resistente");
        } else{
            return;
        }
    }
}

File: bacteria1.c
#include <stdio.h>
#include <string.h>


int main(){

    char entrada[200];
    char comparacao[200];

    scanf ("%s", entrada);
    scanf ("%s", comparacao);

    if (strstr(entrada,comparacao)){

        printf("Resistente");
    } else {
        
        printf ("Não resistente");
    }
    return 0;
}